core/embed/trezorhal/touch/ft6x36.c

```diff
@@ -227,7 +112,6 @@ void touch_power_on(void) {
 }
 
 void touch_power_off(void) {
-  _i2c_deinit();
```

Comment:
I2C is no longer de-initialized when turning off touch, is it intentional? Seems like leaving I2C initialized shouldn't hurt anything and initializing multiple times should be fine.

Seems like `touch_power_off` is only used in prodtest anyway. However there's no `i2c_init` call in prodtest, there probably should be one?

It's also interesting that the bootloader calls `touch_power_on` first then `touch_init` while prodtest does this in reverse.

Reply:
fixed the fmc initialization separately: https://github.com/trezor/trezor-firmware/pull/2927